sklearn/linear_model/ridge.py

```diff
@@ -847,8 +847,10 @@ def fit(self, X, y, sample_weight=None):
         -------
         self : Returns self.
         """
+        alphas = np.asarray(self.alphas)
```

Comment:
For instance, you could test that passing a tuple of alpha, a numpy array of alpha or a list of alpha give similar results.